Islandora-CLAW/Crayfish
# Updating lock files

**GitHub Issue**: Resolves https://github.com/Islandora-CLAW/CLAW/issues/981

# What does this Pull Request do?

Updates compser.lock files so we get the newest crayfish-commons code (and other dependencies).  This should stop the auth errors we're experiencing in https://github.com/Islandora-CLAW/CLAW/issues/981

# How should this be tested?

1. Pull this PR to your Crayfish installation
2. Go into each microservice and run `composer install` to use the new lock file and get new dependencies
3. Check and see if derivatives work again.
 
# Interested parties
@Islandora-CLAW/committers
